spec/jobs/send_webhook_job_spec.rb

```diff
@@ -18,7 +20,7 @@
     end
 
     it 'calls the webhook invoice service' do
-      described_class.perform_now(:invoice, invoice)
+      subject.perform_now(:invoice, invoice)
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)